app/jobs/application_job.rb

```diff
@@ -45,9 +45,6 @@ def capture_exception(error:, extra: {})
   # :nocov:
 
   before_perform do |job|
```

Comment:
```suggestion
  before_perform do
```
not using `job` anymore